docs/sources/talks-and-articles/_index.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Further reading"
+menuTitle: "Further reading"
+description: "Talks and articles about Grafana Mimir"
+weight: 40
+keywords:
+  - Grafana Mimir talks and articles
```

Comment:
```suggestion
  - Grafana Mimir blog posts, podcasts, and videos
```
@brendamuir, how are we standardizing on keywords in metadata?

Reply:
Please input my suggestion and if these need to change, we can change them in a subsequent PR.

Reply:
This is already in.